cobbler.spec

```diff
@@ -264,16 +266,23 @@ mkdir -p %{buildroot}%{tftpboot_dir}/{boot,etc,grub,images{,2},ppc,pxelinux.cfg,
 # systemd
 mkdir -p %{buildroot}%{_unitdir}
 mv %{buildroot}%{_sysconfdir}/cobbler/cobblerd.service %{buildroot}%{_unitdir}
+ln -sf service %{buildroot}%{_sbindir}/rccobblerd
 
 # cobbler-web
 rm %{buildroot}%{_sysconfdir}/cobbler/cobbler_web.conf
 
+# Remove duplicates
+%fdupes -s %{buildroot}%{_prefix}
```

Comment:
This will fail on Debian/Ubuntu (but not Fedora/CentOS). This is also strongly ill-advised, as you are running deduplication across all of `/usr`, which can have bad side-effects.

Reply:
I have removed fdupes and instead will ignore the warning.